cmd/kube-controller-manager/app/autoscaling.go

```diff
@@ -21,11 +21,10 @@ limitations under the License.
 package app
 
 import (
-	"net/http"
-
 	"k8s.io/apimachinery/pkg/runtime/schema"
 	"k8s.io/client-go/dynamic"
 	"k8s.io/client-go/scale"
+	cmcontroller "k8s.io/controller-manager/controller"
```

Comment:
Why cmcontroller, it seems redundant (the c standing for controller)? We can just use controller can't we?

Reply:
Yes. plain `controller` is good enough. Updated.